huntlib/splunk.py

```diff
@@ -105,7 +105,7 @@ def search(self, spl, mode="normal", search_args=None, verbose=False,
                     end_time = end_time.isoformat()
                 search_args["latest_time"] = end_time
 
-        if limit:
+        if limit != None:
```

Comment:
Why change the check to an improper None type check? It's against pep8 standards to check if a variable is "None" is how it currently is. I don't believe this PR would provide value to the repo.

Reply:
Good catch, this should use the `is not` comparison.

Reply:
That's correct! I should've mentioned that part haha it would've been actually helpful, my bad.